src/Form/FoodHeroType.php

```diff
@@ -16,6 +17,9 @@ class FoodHeroType extends AbstractType
     public function buildForm(FormBuilderInterface $builder, array $options)
     {
         $builder
+            ->add('profileImageFile', FileType::class, [
```

Comment:
d'après la doc de Vich
"Now create a form with an imageFile field that uses the **VichImageType type**"